spec/components/dashboard_component_spec.rb

```diff
@@ -241,4 +241,64 @@
       end
     end
   end
+
+  describe "#view_applicants" do
+    let(:organisation) { create(:school) }
+    let(:vacancies) { [vacancy] }
+    let(:vacancy) do
+      create(:vacancy, :published, organisations: [organisation])
+    end
+
+    before do
+      create(:job_application, :status_submitted, vacancy: vacancy)
+      allow(vacancy).to receive(:allow_job_applications?).and_return(uses_either_native_or_uploaded_job_application_form)
+    end
+
+    context "when vacancy uses either native or uploaded job application form" do
+      let(:uses_either_native_or_uploaded_job_application_form) { true }
+
+      context "when the selected type is live" do
```

Comment:
Suggestion: I guess also adding a context/test for `:expired` will increase the test coverage?

Reply:
Sure, done! 🫡